File: Materia/04-Exercicios Diagrama/Exer_01/Que_01_cpp/Veiculo.cpp
#include <string>
using std::string;

#include "Veiculo.h"

string Veiculo::getMarca() const
{
    return marca;
}

string Veiculo::getModelo() const
{
    return modelo;
}

string Veiculo::getPlaca() const
{
    return placa;
}


Cliente Veiculo::getCliente() const
{
    return cliente;
}

void Veiculo::setMarca(string marca)
{
    this->marca = marca;
}

void Veiculo::setModelo(string modelo)
{
    this->modelo = modelo;
}

void Veiculo::setPlaca(string placa)
{
    this->placa = placa;
}

void Veiculo::setCliente(Cliente cliente)
{
    this->cliente = cliente;
}

File: Materia/04-Exercicios Diagrama/Exer_01/Que_01_Java/Veiculo.java
package model;

public class Veiculo {

    private String placa;
    private String marca;
    private String modelo;
    
    private Cliente cliente;

    public Cliente getCliente() {
        return cliente;
    }

    public void setCliente(Cliente cliente) {
        this.cliente = cliente;
    }

    public String getPlaca() {
        return placa;
    }

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }
}
